README.rst

```diff
@@ -2,22 +2,22 @@ Demos
 =====
 A collection of demos using the D-Wave Ocean SDK.
 
-**NOTE** Each demo is stored in its own subdirectory and may have its own
-set of package requirements (for example, `antenna-selection/requirements.txt
-<antenna-selection/requirements.txt>`_).
+**NOTE** Each demo is stored in its own submodule and may have its own
+set of package requirements (e.g. `antenna-selection/requirements.txt
+<https://github.com/dwave-examples/antenna-selection/blob/master/requirements.txt>`_).
 
 Setup Instructions
 ------------------
-As each demo may have a different set of package requirements, you may want to
-set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
+As each demo may have a different set of package requirements, you may
+want to set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
 to contain said packages and run your demo within that environment.
 
 Package Installation
 ~~~~~~~~~~~~~~~~~~~~
 ::
 
-  git clone https://github.com/dwavesystems/demos.git
-  cd demos/<desired-demo>
+  git clone https://github.com/dwave-examples/<desired-demo>.git
```

Comment:
This is confusing. We're talking about collection of demos in this repo, and then we instruct the user to clone from a different account...

I would:
a) be explicit `demos.git` is just a transitional thing, until we fully switch to `dwave-examples`, or
b) clone with submodules included from `demos.git` (either with `--recursive`, or with separate `git submodule init/update`)

Reply:
Choosing option (b) because it provides the same functionality as the `demos` prior to the submodules change

Reply:
Actually choosing (c), all of the above. I'll make a note that this is transitional and that we will eventually only have `dwave-examples` for the code examples